src/_locales/en_US/messages.json

```diff
@@ -211,6 +211,30 @@
         "message": "Export user data",
         "description": ""
     },
+    "reset": {
+        "message": "Reset",
+        "description": ""
+    },
+    "reset_user_data": {
+        "message": "Reset browsing data",
+        "description": ""
+    },
+    "reset_data_confirm": {
+        "message": "This will delete all browsing data that Privacy Badger has collected and restore the extension to its default state. Some trackers which are currently blocked may be allowed through afterwards.",
+        "description": ""
+    },
+    "clear_all": {
+        "message": "Clear all",
+        "description": ""
+    },
+    "clear_all_data": {
+        "message": "Clear all tracker data",
```

Comment:
"Remove all tracking domains" instead? "Tracking domains" is an existing thing, "tracker data" is new.